packages/app/obojobo-document-engine/src/scripts/oboeditor/components/node/components/insert-menu.js

```diff
@@ -68,7 +68,7 @@ class InsertMenu extends React.PureComponent {
 
 			// Move left/up through the insert menu
 			// Left is logical for sighted keyboard users; up is a standard
-			// navigation principle for submenus
+			// navigation principle for submenus, and therefore logical to screenreaders
 			case 'ArrowLeft':
 			case 'ArrowUp':
```

Comment:
This doesn't seem to be working for me - If I'm in file and moving up and down the list, then press right, nothing happens. Unless this behavior is intended, but my expectation was for it to move to the next menu fwiw.

Reply:
In the File at the top of the editor, only the up/down works, since it's a vertically displayed menu (I couldn't figure out how to get the left/right working except as open/close because of the way the components are rendered).  In the inline insert menus, sighted users might expect to be able to use the left/right arrows since the menu is horizontal, so both the left/right and up/down scroll through the menu, and Escape closes the menu instead